app/models/podcast.rb

```diff
@@ -55,20 +55,20 @@ def self.by_prx_series(series)
     Podcast.find_by(prx_uri: series_uri)
   end
 
+  def self.enclosure_template_default
+    "https://#{ENV['DOVETAIL_HOST']}{/podcast_id,feed_slug,guid,original_basename}{feed_extension}"
```

Comment:
Note the addition of a `feed_extension`, to match when the audio for this feed is different